gui/slick/views/displayShow.mako

```diff
@@ -353,7 +353,9 @@
             <th class="row-seasonheader displayShowTable" colspan="13" style="vertical-align: bottom; width: auto;">
                 <h3 style="display: inline;"><a name="season-${epResult["season"]}"></a>${("Specials", "Season " + str(epResult["season"]))[int(epResult["season"]) > 0]}
                 <!-- @TODO: port the season scene exceptions to angular -->
+                % if not any([i for i in sql_results if epResult['season'] == i['season'] and int(i['status']) == 1]):
```

Comment:
Maybe this is the best option we have. But did wanted to note that this does not guarantee that the season has finished. It only says, that medusa indexed unaired episodes. What if the season hasn't been finished yet, but the indexer also doesn't show information on new eps?